Vic2ToHoI4/Data_Files/blankMod/output/common/scripted_localisation/00_scripted_localisation_FR_loc.txt

```diff
@@ -2,3 +2,2591 @@
 ###   Scripted localization for French by Nicolas Fouqué   ###
 ##############################################################
 
+#################################################
+###   Political Parties / Partis politiques   ###
+#################################################
+# Sidenote: All political parties now fallback under masculine plural, like we're referring to the people inside it.
+# It is easier than having to deal with all Vic2 and HoI4 possible parties and scripted triggers
+
+### Ruling Parties / Partis au pouvoir ###
+
+### "The ruling party is [the] NSDAP/IKL/Misr al-Fatah/UNF/Armenian Legion/Fire Horses." => "Le parti au pouvoir est [le] NSDAP/[] Misr al-Fatah/[l']IKL/[la] Légion arménienne/[les] Chevaux de feu" (lower case / minuscule)
+defined_text = {
+	name = GetLeLaLPartiPouv
+	text = {
+		trigger = {
+      always = yes
```

Comment:
Replace the four spaces here with three tabs. Do so for all cases of ```always = yes``` using replace all

Reply:
Done